internify/src/components/molecules/styles/Notes.css

```diff
@@ -0,0 +1,5 @@
+.notes_row {
+  margin-bottom: 1%;
+  font-family: Helvetica;
```

Comment:
You shouldn't need to specify the font-family all the time. I've set it in the `body` styles in `App.css`. Was there an issue with the CSS?